OneCommunityGlobal/HighestGoodNetworkApp
# Shree create blue square reason scheduler forall fe

# Description
![bug_desc_1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/41186831/a2a0441b-04a3-44e3-b20e-4a86fdc074f8)

![bug_desc_2](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/41186831/2f202753-1713-4704-be40-b63769835b3e)

## Related PRS (if any):
To test this frontend PR you need to checkout the [#614](https://github.com/OneCommunityGlobal/HGNRest/pull/614) backend PR.

## Main changes explained:
- Update file BlueSquareLayout.jsx to add conditions on accessing the scheduler when there are no blue squares present.
- Update file ScheduleReasonModal.jsx to change text,add buttons, create flag to track   reason edits, add placeholder, title.
- Create file ScheduleReasonModal.css

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally. The frontend is up and running.
3. To save the reason - either add a new reason or edit an existing one.
4. Look and focus only on the text changes and FAQ button (no action/text is expected), please ignore other css. 

## Screenshots or videos of changes:
Before :
![Before](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/41186831/c647e180-6deb-47b6-9044-d05b9a6cc69b)

After :
![After](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/41186831/5f7d29a3-8c8e-4af3-aa38-74be919b8440)
